src/main/kotlin/app/ui/profilePreview/View/ProfilePreview.kt

```diff
@@ -0,0 +1,162 @@
+package app.ui.profilePreview.View
+
+import app.UIColorsObject.Colors
+import app.ui.userCreation.*
+import app.ui.profilePreview.ViewModel.ProfilePreviewViewModel
+import app.widgets.profileIcon.ProfileIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import io.reactivex.rxkotlin.subscribeBy
+import io.reactivex.subjects.PublishSubject
+import javafx.geometry.Pos
+import javafx.scene.Cursor
+import javafx.scene.effect.DropShadow
+import app.ui.welcomeScreen.WelcomeScreen
+import tornadofx.*
+import app.widgets.WidgetsStyles
+
+class ProfilePreview : View() {
+
+    private val viewModel = ProfilePreviewViewModel()
+    var iconHash = viewModel.userIconHash
+    var onClickNext = viewModel.onClickNext
+    var onClickRedo = viewModel.onClickRedo
+    var audioListened =viewModel.audioListened
```

Comment:
these values are just going to end up out of sync with your viewmodel though?

Reply:
not if you inject the view model though right?